app/assets/stylesheets/redesign/_card-action.scss

```diff
@@ -187,7 +185,7 @@ $action-bar-wrapper-height-small: 60px;
 
 .action-bar-wrapper{
   bottom:  0px;
-  background: #fff;
+  background: rgba(255,255,255,.9);
```

Comment:
Buttons on action bar were unnecessarily transparent. This just makes the action bar background color transparent, without affecting the content.